packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -119,14 +124,14 @@ export class LocalSigningManager implements SigningManager {
   /**
    * @hidden
    */
-  private constructor(accounts: PrivateKey[]) {
+  private constructor(arg: { accounts: PrivateKey[]; curveType: Curve }) {
```

Comment:
(style)
```suggestion
  private constructor(arg: { accounts: PrivateKey[]; curveType: Curve }) {
  const { accounts, curveType } = args
```
then remove `arg.`

We use destructuring quite a bit, so it will be a bit more consistent with the SDK style. No behavior change though.